nimdoc/testproject/subdir/subdir_b/utils.nim

```diff
@@ -115,3 +115,8 @@ proc f*(x: G[int]) =
 proc f*(x: G[string]) =
   ## See also `f(G[int])`_.
   discard
+
+## Ref. `[]`_ is the same as `proc \`[]\`(G[T])`_ because there are no
```

Comment:
would this work to avoid quoting?
```nim
## See `proc \`$\`(int)`_
## See ``proc `$`(int)``_
```
the 2nd is more readable because it's easier to copy paste from a declaration

Reply:
Not now :-(

In original RST it would not even require escaping, but our implementation was tweaked in favor of Markdown-like inline markup rules, which allow to put the ending backtick anywhere and also put starting backtick anywhere. So with single-backtick syntax we are bound to escaping of backticks.

Regarding double-backticks: the syntax is sane and I like it but it's not standard and we do not support it, though it would probably be easy to implement. I think anything like this will be left for future PRs anyway to avoid conflation here.

I agree it's ugly, that's why to make it less painful I added possibility to ref. operators the short way `` `$`_`` (instead of a long way like
```
`\`$\``_
```
)
but for any compound anchors one still has to use escaping:
```
`proc \`$\``_
```